test/cypress/integration/settings/settings-index.spec.js

```diff
@@ -0,0 +1,43 @@
+describe('Settings page', () => {
+  const userExternalId = 'cd0fa54cf3b7408a80ae2f1b93e7c16e'
+  const gatewayAccountId = 'DIRECT_DEBIT:42'
+  const serviceName = 'A Direct Debit Service'
+
+  describe('should redirect to API keys if Direct Debit gateway', () => {
+    beforeEach(() => {
+      cy.task('setupStubs', [
+        {
+          name: 'getUserSuccess',
+          opts: {
+            external_id: userExternalId,
+            service_roles: [{
+              service: {
+                gateway_account_ids: [gatewayAccountId],
+                name: serviceName
+              }
+            }]
+          }
+        },
+        {
+          name: 'getDirectDebitGatewayAccountSuccess',
+          opts: {
+            gateway_account_id: gatewayAccountId
+          }
+        }
+      ])
+    })
+
+    it('should show it is disabled', () => {
```

Comment:
what is disabled?

Reply:
Oh lol left over from test I copied will update words